content/blog/gatsby-tina-101.md

```diff
@@ -582,18 +596,19 @@ const Layout = props => (
   </React.Fragment>
 )
 
+// Export Layout with CreatePostPlugin
 export default withPlugin(Layout, CreatePostPlugin)
 ```
 
-`withPlugin` is another higher-order component that adds a plugin to the Tina sidebar when the component it wraps is being used. Since my `Layout` component is used for every page in _Tyra_, content authors can easily add posts from anywhere!
+`withPlugin` is another Higher-Order Component — it adds a plugin to the Tina sidebar when the component it wraps is being rendered. Since `Layout` is at the root of every page in _Tyra_, content authors can add posts from anywhere in the site!
```

Comment:
```suggestion
`withPlugin` is another Higher-Order Component — it adds a plugin to the Tina sidebar when the component it wraps is being rendered. Since `Layout` is at the root of every page in _Tyra_, content authors can add posts from anywhere on the site!
```